aarch64_linux/aarch64_wheel_ci_build.py

```diff
@@ -103,7 +103,7 @@ def update_wheel(wheel_path) -> None:
     os.system(f"unzip {wheel_path} -d {folder}/tmp")
     libs_to_copy = [
         "/usr/local/cuda/extras/CUPTI/lib64/libcupti.so.12",
-        "/usr/local/cuda/lib64/libcudnn.so.8",
+        "/usr/local/cuda/lib64/libcudnn.so.9",
```

Comment:
out of curiosity, do we plan to carry both the legacy API and new API here so that we can migrate pytorch over to the new graph API? this seems like the size impact would roughly double which as things currently stand ... its already quite a lot in terms of overall final artifact size 

ref https://docs.nvidia.com/deeplearning/cudnn/latest/api/overview.html

Reply:
Good point and I think this question applies to x86 as well. 
This PR was created not considering the size impact and ship both legacy and new API .so files, for both x86 and arm. 

cc @ptrblck @eqy @malfet @atalman @tinglvv for additional inputs